pytorch_lightning/core/lightning.py

```diff
@@ -87,7 +92,7 @@ def forward(self, x):
             log.info(*args, **kwargs)
 
     @abstractmethod
-    def forward(self, *args, **kwargs):
+    def forward(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
not sure if these are typed... but probably yes...
https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs

Reply:
yes, i am also not sure, i am not a typing expert yet :)

Reply:
@hadim any thoughts?

Reply:
For `args` and `kwargs` I would not bother adding type. IMO this is the only situation where not add types.

Reply:
Except for specific situations where you want for example to force `args` to be `List[int]` or something similar...

Reply:
got it. makes sense :)